### ISSUE TITLE
Strange recognition

### ISSUE THREAD

**Issue by lily-ab** [0x5eba/Skin-Cancer-Segmentation#5]
Hello,
I wanted to repeat the learning experience of this neural network and after 90 epoches it still can't recognize anything correctly.
So, could you answer a few questions, please?
1) Have you sorted your dataset somehow and how many images have been there?
2) How many epoches have you used?
3) Which values or learning rate have you used?

I've gotten most results like these on the screenshots. What can I do wrong, why do I get this?
![default](https://user-images.githubusercontent.com/33357724/52177948-d53a3f80-27e1-11e9-8cfb-e16843c7dc9d.png)
![default](https://user-images.githubusercontent.com/33357724/52177958-e5eab580-27e1-11e9-8305-c2151596fb40.png)
![default](https://user-images.githubusercontent.com/33357724/52177961-eedb8700-27e1-11e9-9204-e3b63babe6ae.png)

**Comment by maiduyanh1705** [comment]
same issues, anyone can help us?

**Comment by hungdaica77** [comment]
you train 50gb data?

**Comment by HoneyDevoli** [comment]
> 
> 
> you train 50gb data?

no, I was compressing pictures

**Comment by hungdaica77** [comment]
> > you train 50gb data?
> 
> no, I was compressing pictures

i trained 50gb data on 90 epoch but it's not detect malignat
